### ISSUE TITLE
Name resolution problem with AWS Ubuntu 18.04

### ISSUE THREAD

**Issue by manis-ai** [putsi/privatecollaborator#12]
Hi Putsi,

Thanks for this awesome script!

I tried running this on a fresh AWS Ubuntu 18.04. After running install.sh, name resolution stopped working. Commenting out these lines from install.sh seem to solve the issue:

systemctl disable systemd-resolved.service
systemctl stop systemd-resolved
rm -rf /etc/resolv.conf
echo "nameserver 1.1.1.1" > /etc/resolv.conf
echo "options edns0" >> /etc/resolv.conf
echo "search eu-north-1.compute.internal" >> /etc/resolv.conf

**Comment by manis-ai** [comment]
You can change the default AWS nameserver by creating a file 99-custom-dns.yaml located at /etc/netplan:
`sudo vim /etc/netplan/99-custom-dns.yaml`

Contents of the file:

```
network:
    version: 2
    ethernets:
        ens5:         
            nameservers:
                    addresses: [8.8.8.8, 1.1.1.1]
            dhcp4-overrides:
                    use-dns: false
```

(Replace ens5 with your network interface name. )

After this reboot the machine or run:
`netplan apply`

To see, if new nameservers are applied correctly, run:
`systemd-resolve --status`

This is explained in this AWS knowledge base article:
https://aws.amazon.com/premiumsupport/knowledge-center/ec2-static-dns-ubuntu-debian/

**Comment by putsi** [comment]
```
systemctl disable systemd-resolved.service
systemctl stop systemd-resolved
rm -rf /etc/resolv.conf
echo "nameserver 1.1.1.1" > /etc/resolv.conf
```

The above lines were needed previously because systemd-resolved bound the port 127.0.0.1:53 and that caused issues. 
I think old version of certbot or old version of Burp Collaborator required port 53 on localhost which broke things and needed systemd-resolved to be disabled.

I think it might be possible to remove that hack now, but I'll have to test if it actually is possible.
